load.php

```diff
@@ -255,6 +256,11 @@ function perflab_can_load_module( $module ) {
 		return true;
 	}
 
+	// Check if can load return an error.
+	if ( is_wp_error( $module() ) ) {
+		return $module();
+	}
+
 	// Call the closure to determine whether the module can be loaded.
 	return (bool) $module();
```

Comment:
Nit: just call the function once

```suggestion
	// Call the closure to determine whether the module can be loaded.
	$result = $module();

	if ( is_wp_error( $result ) ) {
		return $result;
	}

	return (bool) $result;
```

Reply:
Missed it. Thanks for pointing.